cc/tflite_micro/oak/libc/bsearch.c

```diff
@@ -0,0 +1,105 @@
+/*	$NetBSD: bsearch.c,v 1.15 2012/03/04 20:01:45 christos Exp $	*/
+
+/*
+ * Copyright (c) 1990, 1993
+ *	The Regents of the University of California.  All rights reserved.
```

Comment:
This is not our code, so it must go under third_party. BTW, where was it taken from, and how can we keep it up to date in the future? Just a readme with a link may be enough for now. Alternatively, was it not possible to include it with http_archive from the bazel workspace?

Reply:
Help me clarify what I should put in third_party complying with oak open sourcing policies
.
Should I put third-party sources that are modified by us in third_party?
For libc, some sources from us directly, some from third-party w/o modifications, some from third-party w/ modifications, how should I organize them in oak repo?

Btw, android bionic libc open source, lots of third-party sources not written by android are still organized together with android own sources, that makes me feel more readable, searchable and well organized as lib users.

Reply:
Btw, bsearch is from android bionic libc and I modified it to remove header dependencies.

Reply:
I eventually moved it to third_party.